packages/utils/src/carequality/patient-discovery-statistics.ts

```diff
@@ -0,0 +1,79 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// // keep that ^ on top
+import { patientDataFromResource } from "@metriport/core/external/carequality/pd/process-inbound-pd";
+import { MPIMetriportAPI } from "@metriport/core/mpi/patient-mpi-metriport-api";
+import { inboundPatientResourceSchema } from "@metriport/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses";
+import { QueryTypes, Sequelize } from "sequelize";
+import z from "zod";
+import { getEnvVarOrFail } from "../../../api/src/shared/config";
+
+const apiUrl = getEnvVarOrFail("API_URL");
+const mpi = new MPIMetriportAPI(apiUrl);
+
+const cxId = getEnvVarOrFail("CX_ID");
+const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+const dbCreds = JSON.parse(sqlDBCreds);
+const sequelize = new Sequelize(dbCreds.dbname, dbCreds.username, dbCreds.password, {
+  host: dbCreds.host,
+  port: dbCreds.port,
+  dialect: dbCreds.engine,
+  logging: false,
+});
+
+const rowWithDataSchema = z.object({
+  status: z.string(),
+  data: z
+    .object({
+      patientResource: inboundPatientResourceSchema.optional(),
+      timestamp: z.string(),
+    })
+    .optional(),
+});
+
+const patientId = "";
+
+async function main() {
+  const query = `
+  SELECT * FROM patient_discovery_result
+  WHERE data->>'cxId'='${cxId}'
```

Comment:
Let's move this to [Bind parameters](https://sequelize.org/docs/v6/core-concepts/raw-queries/#bind-parameter), please. Those make it much more secure as they prevent SQL injection attacks - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#c85966880dbb409fb169da526918c8cf).